test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/BasicTagHelpers.RemoveTagHelper.cshtml

```diff
@@ -1,5 +1,5 @@
-﻿@addTagHelper something, nice
-@removeTagHelper doesntmatter, nice
+﻿@addTagHelper "something, nice"
```

Comment:
Do this for a few of the design time code generation tests so we can validate line mappings. Should be one for each, add, remove, prefix